src/lib/OpenEXR/ImfIO.h

```diff
@@ -246,14 +237,14 @@ struct StreamIO
 
 struct CharPtrIO
 {
-    static void
+    static inline void
```

Comment:
I think any method fully defined inside a struct or class declaration is implicitly `inline`.

Reply:
yes